x-pack/plugins/lens/public/datasources/form_based/operations/definitions/formula/editor/formula_help.tsx

```diff
@@ -185,122 +211,77 @@ max(system.network.in.bytes, reducedTimeRange="30m")
-    }),
-    items: [],
-  });
-
-  const mathFns = mathOperations.sort().map((key) => {
-    const [description, examples] = tinymathFunctions[key].help.split(`\`\`\``);
-    return {
-      label: key,
-      description: description.replace(/\n/g, '\n\n'),
-      examples: examples ? `\`\`\`${examples}\`\`\`` : '',
-    };
-  });
-
-  helpGroups[4].items.push(
-    ...mathFns.map(({ label, description, examples }) => {
-      return {
-        label,
-        description: (
-          <>
-            <h3>{getFunctionSignatureLabel(label, operationDefinitionMap)}</h3>
+  helpGroups.push(
+    createNewSection(
+      i18n.translate('xpack.lens.formulaDocumentation.mathSection', {
+        defaultMessage: 'Math',
+      }),
+      i18n.translate('xpack.lens.formulaDocumentation.mathSectionDescription', {
+        defaultMessage:
+          'These functions will be executed for reach row of the resulting table using single values from the same row calculated using other functions.',
+      }),
 
-            <Markdown markdown={`${description}${examples}`} />
-          </>
-        ),
-      };
-    })
+      mathOperations,
+      operationDefinitionMap
+    )
   );
 
-  helpGroups.push({
-    label: i18n.translate('xpack.lens.formulaDocumentation.comparisonSection', {
-      defaultMessage: 'Comparison',
-    }),
-    description: i18n.translate('xpack.lens.formulaDocumentation.comparisonSectionDescription', {
-      defaultMessage: 'These functions are used to perform value comparison.',
-    }),
-    items: [],
-  });
-
-  const comparisonFns = comparisonOperations.sort().map((key) => {
-    const [description, examples] = tinymathFunctions[key].help.split(`\`\`\``);
-    return {
-      label: key,
-      description: description.replace(/\n/g, '\n\n'),
-      examples: examples ? `\`\`\`${examples}\`\`\`` : '',
-    };
-  });
-
-  helpGroups[5].items.push(
-    ...comparisonFns.map(({ label, description, examples }) => {
-      return {
-        label,
-        description: (
-          <>
-            <h3>{getFunctionSignatureLabel(label, operationDefinitionMap)}</h3>
+  helpGroups.push(
+    createNewSection(
+      i18n.translate('xpack.lens.formulaDocumentation.comparisonSection', {
+        defaultMessage: 'Comparison',
+      }),
+      i18n.translate('xpack.lens.formulaDocumentation.comparisonSectionDescription', {
+        defaultMessage: 'These functions are used to perform value comparison.',
+      }),
+
+      comparisonOperations,
+      operationDefinitionMap
+    )
+  );
 
-            <Markdown markdown={`${description}${examples}`} />
-          </>
-        ),
-      };
-    })
+  helpGroups.push(
+    createNewSection(
+      i18n.translate('xpack.lens.formulaDocumentation.constantsSection', {
+        defaultMessage: 'Kibana context',
+      }),
+      i18n.translate('xpack.lens.formulaDocumentation.constantsSectionDescription', {
+        defaultMessage: 'Functions used to retrieve Kibana context variables.',
```

Comment:
The other Formula sections in this window include thoroughly broken down definitions. 

How about:
`These functions are used to retrieve Kibana context variables, which are... and help you to....`

Reply:
I've revisited the copy here. Maybe we could discuss a bit and have a consistent copy for the entire section with a follow up PR here?